tests/aws/templates/ec2_instance_keypair.yml

```diff
@@ -0,0 +1,17 @@
+Resources:
+  NewKeyPair:
+    Type: 'AWS::EC2::KeyPair'
+    Properties:
+      KeyName: !Sub 'keypair-${AWS::StackName}'
+  Ec2Instance:
+    Type: 'AWS::EC2::Instance'
+    Properties:
+      ImageId: "ami-0e9107ed11be76fde"
+      KeyName: !Ref NewKeyPair
```

Comment:
Any particular reason why the instance is included in the template here?

Reply:
I wrongly thought there were no deployment test for ec2 instances. so I rewrote the test a little just to assert the deployment and deletion of the keypair